plugins/experimental/sslheaders/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 #######################
 
 add_library(sslhdr STATIC expand.cc util.cc)
-target_link_libraries(sslhdr PRIVATE OpenSSL::SSL)
+target_link_libraries(sslhdr PUBLIC OpenSSL::SSL)
```

Comment:
I think we should keep this `PRIVATE` and link `test_sslhdr` to `OpenSSL::SSL` instead. The missing headers are included directly by the test; it's not a transitive dependency.